conf/defaults.ini

```diff
@@ -264,6 +264,9 @@ password_hint = password
 # Default UI theme ("dark" or "light")
 default_theme = dark
 
+# Default timezone
```

Comment:
Should we add a reference to what timezones are allowed? Should the timezone be checked and a warning logged? It defaults to browser if an invalid timezone is given, currently